docs/end-user/policies/envbinding.md

```diff
@@ -1,3 +1,123 @@
 ---
 title: Env-Binding
----
\ No newline at end of file
+---
+
+This documentation will introduce how to use env-binding to provide differentiated configuration and environment scheduling strategies for Applications.
+
+## Background
+
+In daily development, users often deploy Application to different environments. For example, debug the Application in the development environment, and deploy Application in the production environment to provide services. For different environments, Application require differentiated configurations.
+
+## How to
+
+Deploy the following Application:
+
+```yaml
+# app.yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: policy-demo
+  namespace: default
+spec:
+  components:
+    - name: nginx-server
+      type: webservice
+      properties:
+        image: nginx:1.21
+        port: 80
+
+  policies:
+    - name: demo-policy
+      type: env-binding
+      properties:
+        clusterManagementEngine: single-cluster
+        envs:
+          - name: test
+            patch:
+              components:
+                - name: nginx-server
+                  type: webservice
+                  properties:
+                    image: nginx:1.20
+                    port: 80
+            placement:
+              namespaceSelector:
+                name: test
+                
+    workflow:
+      steps:
+        - name: deploy-test-env
+          type: multi-env
+          properties:
+            # Specify the policy name
+            policy: demo-policy
+            # Specify the env name in the policy
+            env: test
+```
+
+User wants to deploy the Application `policy-demo` to the namespace `test`(test environment) and modify some parameters in the component `nginx-server` to adapt to the test environment.
```

Comment:
```suggestion
This will deploy the Application `policy-demo` to the namespace `test`(test environment), and patch the parameters in the component `nginx-server` to adapt to the test environment.
```